ui/app-start-intent/src/main/java/de/danoeh/antennapod/ui/appstartintent/MainActivityStarter.java

```diff
@@ -51,4 +53,10 @@ public MainActivityStarter withAddToBackStack() {
         intent.putExtra(EXTRA_ADD_TO_BACK_STACK, true);
         return this;
     }
+
+    public MainActivityStarter withFragmentLoaded(String fragmentName) {
+        intent.putExtra(EXTRA_FRAGMENT_TAG, fragmentName);
+        intent.putExtra(EXTRA_OPEN_DRAWER, true);
```

Comment:
These should be two different methods. One that loads a fragment and one that opens the drawer.